doc/man3/ossl_param_get_int.pod

```diff
@@ -0,0 +1,173 @@
+=pod
+
+=head1 NAME
+
+ossl_param_get_int8, ossl_param_get_uint8, ossl_param_get_int16,
+ossl_param_get_uint16, ossl_param_get_int32, ossl_param_get_uint32,
+ossl_param_get_int64, ossl_param_get_uint64, ossl_param_get_int,
+ossl_param_get_uint, ossl_param_get_char, ossl_param_get_uchar,
+ossl_param_get_short, ossl_param_get_ushort, ossl_param_get_long,
+ossl_param_get_ulong, ossl_param_get_long_long, ossl_param_get_ulong_long,
+ossl_param_get_size_t, ossl_param_get_float, ossl_param_get_double,
+ossl_param_get_BN, ossl_param_set_int8, ossl_param_set_uint8,
+ossl_param_set_int16, ossl_param_set_uint16, ossl_param_set_int32,
+ossl_param_set_uint32, ossl_param_set_int64, ossl_param_set_uint64,
+ossl_param_set_int, ossl_param_set_uint, ossl_param_set_char,
+ossl_param_set_uchar, ossl_param_set_short, ossl_param_set_ushort,
+ossl_param_set_long, ossl_param_set_ulong, ossl_param_set_long_long,
+ossl_param_set_ulong_long, ossl_param_set_size_t, ossl_param_set_float,
+ossl_param_set_double, ossl_param_set_BN
+- OSSL_PARAMS helpers
```

Comment:
Public API, so should start with uppercase `OSSL_PARAM_`

Reply:
Okay.

Reply:
Maybe in the documentation too?  :wink:

Reply:
Doh!  Done.